Source/EPUBCore/FRMediaType.swift

```diff
@@ -9,21 +9,21 @@
 import UIKit
 
 /**
-MediaType is used to tell the type of content a resource is.
+ MediaType is used to tell the type of content a resource is.
 
-Examples of mediatypes are image/gif, text/css and application/xhtml+xml
-*/
+ Examples of mediatypes are image/gif, text/css and application/xhtml+xml
+ */
 struct MediaType {
-    var name: String
-    var defaultExtension: String!
-    var extensions: [String]!
-    
+    var name                : String
+    var defaultExtension    : String!
+    var extensions          : [String]!
```

Comment:
I am/We are not using `Forced Unwrapping` there. Here at the SmartMobileFactory this this strictly forbidden. Those unwrappings have been implemented before we started any change in the library.

On this pull request, while implementing the new feature I personally removed many `Forced Unwrapping` in favour of `Optional` ones. But the cost of such migration is pretty heavy as it takes a lot of times and could also create side-effect bugs.